lib/utils/app_list.dart

```diff
@@ -139,3 +142,10 @@ Application get fallbackApp {
   return Application(
       app: ErrorWindow(), name: "Error", packageName: "io.dahlia.error");
 }
+
+extension appWebExtension on Application {
```

Comment:
follow the name convention for dart class/extension names, it should be AppWebExtension